packages/mobx-react/__tests__/__snapshots__/observer.test.tsx.snap

```diff
@@ -35,23 +37,7 @@ exports[`Redeclaring an existing observer component as an observer should log a
 }
 `;
 
-exports[`SSR works #3448 1`] = `
```

Comment:
Was not supposed to warn. Warning was caused by calling `disableStaticRendering` before `unmount` in the test. Therefore `componentWillUnmount` used logic for non-static rendering.